conf/recipe.yaml

```diff
@@ -55,3 +55,24 @@ Manifests:
           exit 100
   - Platform:
       os: windows
+    Artifacts:
+      - URI: s3://use-local-provided-artifact/custom.zip
+        Unarchive: zip
+        Permission:
+          execute: ALL
+          read: ALL
+    Lifecycle:
+      bootstrap:
+        RequiresPrivilege: true
+        script: |-
+
+          powershell -command "Set-StrictMode -Version Latest;"^
```

Comment:
you sure that this works? How did you test it?
My experience is that multi-line commands won't work (despite escaping with `^`).

Reply:
Yes I've tested like the description says